### DISCUSSION TITLE
Welcome to svgpathtools Discussions!

### DISCUSSION THREAD

**Discussion by mathandy** [mathandy/svgpathtools#134] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by soyasis** [comment]
Hi! First of all thanks to the community for sharing and maintaining such an awesome tool! 🥳
I have a specific problem which I'm sure has a solution, but struggling with it and maybe someone has a good hint: 

**Goal:** 
I am trying to match a path across different `.svg` files (in different positions and without using attributes like Id)
  
**Input:**  (see attached files)
- `arrow.svg` - contains single element
- `find-arrow.svg` - contains multiple elements  

**Desired output:**
- List of paths (or indexes of `Paths` list) in `svg2paths2('find-arrow.svg')` that match the path in `arrow.svg`. Looking to do the matching only based on the path across different viewBoxes, so colors and other attributes are not considered.

Trying to first simplify problem with this basic examples, but the idea is then to do the same for more elaborate paths.
Any suggestions will be highly appreciated! 😇

![arrow](https://user-images.githubusercontent.com/38328023/205081581-a0cb5b84-7fd8-4a98-a45b-0004ca29f555.svg)
![find-arrow](https://user-images.githubusercontent.com/38328023/205085572-b4d5a47e-e8f7-475e-8317-644ff764dbf6.svg)

**Reply by soyasis** [reply]
**UPDATE:**
I created a [Jupyter Notebook](https://github.com/soyasis/simple-svg-tests/blob/main/example-match-arrows.ipynb) solving the above problem, definitely not the most elegant way, but so far seems to work for simple figures. Feedback would be highly appreciated, still **a lot** to learn here! 😇
Gonna experiment now with groups of paths.
